Makefile

```diff
@@ -173,7 +173,7 @@ bots:
 # checkout Cockpit's test API; this has no API stability guarantee, so check out a stable tag
 # when you start a new project, use the latest release, and update it from time to time
 test/common:
-	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 228
+	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git b7fc7bd5e517659c45531dcf84505902a17ec9e8
```

Comment:
That's a little wild, but indeed on second look nicer than ading it to the test's setUp() temporarily. :-)

Reply:
yea and releasing 229.1 seems like a bit much :p